InternshipBE/DAL/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DAL.Interfaces
+{
+	public interface IRepository<T>
+	{
+		public void Create(T model);
```

Comment:
1. rename to entity
2. spaces between methods, properties...
3. line 13 - empty line